client/my-sites/domains/domain-management/transfer/transfer-out/unlocked.jsx

```diff
@@ -62,81 +99,193 @@ class Unlocked extends React.Component {
 		const options = {
 			siteId: this.props.selectedSite.ID,
 			domainName: this.props.selectedDomainName,
 			unlock: false,
 			disablePrivacy: false
 		};
 
+		this.setState( { submitting: true } );
+
 		requestTransferCode( options, ( error ) => {
 			this.setState( { submitting: false } );
+			if ( ! error ) {
+				this.setState( { sent: true, } );
+			}
 			displayRequestTransferCodeResponseNotice( error, getSelectedDomain( this.props ) );
 		} );
-	};
+	}
+
+	renderPendingTransferBody() {
+		const { translate } = this.props;
+		return ( <div>
+			<p>{ translate( 'Your domain is pending transfer.' ) }</p>
+		</div> );
+	}
+
+	renderManualTransferBody() {
+		const { translate } = this.props;
+		const { sent } = this.state;
+
+		return ( <p>{
+			translate( 'The registry for your domain requires a special process for transfers. ' )
+		} { sent
+			? translate( 'Our Happiness Engineers have been notified about ' +
+				'your transfer request and will be in touch shortly to help ' +
+				'you complete the process.' )
+			: translate( 'Please request an authorization code to notify our ' +
+				'happiness engineers of your intention.' )
+		}</p> );
+	}
+
+	renderAuthorizationCodeBody() {
+		const { translate } = this.props;
+		const { submitting, sent } = this.state;
+
+		const sentStatement = sent &&
+			translate( 'We have sent the transfer authorization code to the ' +
+				'domain registrant\'s email address.' ) +
+			' ';
+		return ( <div>
+			{ ! ( submitting || sent ) &&
+				<p>{ translate( 'Please press the button to request a transfer authorization code.' ) }</p>
+			}
+			<p>
+				{ sentStatement }
+				{ translate( 'You must provide your new registrar with your ' +
+					'domain name and transfer code to complete the transfer process.' )
+				}
+			</p>
+		</div> );
+	}
 
 	render() {
 		const { translate } = this.props;
-		const { privateDomain, hasPrivacyProtection, manualTransferRequired, pendingTransfer } = getSelectedDomain( this.props );
-		let domainStateMessage = translate( 'Your domain is unlocked to prepare for transfer.' );
+		const { submitting, } = this.state;
+		const domain = getSelectedDomain( this.props );
+		const {
+			privateDomain,
+			hasPrivacyProtection,
+			domainLockingAvailable,
+		} = domain;
+		const privacyDisabled = hasPrivacyProtection && ! privateDomain;
 
-		if ( pendingTransfer ) {
-			domainStateMessage = translate( 'Your domain is pending transfer.' );
-		} else if ( hasPrivacyProtection && ! privateDomain ) {
-			domainStateMessage = translate( 'Your domain is unlocked and Privacy Protection has been disabled' +
-				' to prepare for transfer.' );
+		let domainStateMessage;
+		if ( domainLockingAvailable && privacyDisabled ) {
+			domainStateMessage = translate( 'Your domain is unlocked and ' +
+				'Privacy Protection has been disabled to prepare for transfer.' );
+		} else if ( privacyDisabled ) {
+			domainStateMessage = translate( 'Privacy Protection for your ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Your domain is unlocked and Privacy Protection has been disabled to prepare for transfer.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).